platform/x11/os_x11.cpp

```diff
@@ -78,19 +78,20 @@
 #include <X11/XKBlib.h>
 
 int OS_X11::get_video_driver_count() const {
-	return 1;
+
+	return 2;
 }
 
 const char *OS_X11::get_video_driver_name(int p_driver) const {
-	String driver_name = GLOBAL_GET("rendering/quality/driver/driver_name");
 
-	if (driver_name == "GLES2") {
+	if (p_driver == GLES_2)
 		return "GLES2";
-	}
-	return "GLES3";
+	else
```

Comment:
isn't it better to remove `else` here?

Reply:
Well while we're at it, we could even use a switch, as we'll have `VULKAN` in the enum at some point :)